src/main/java/edu/ie3/datamodel/io/factory/result/CongestionResultFactory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.factory.result;
+
+import static edu.ie3.util.quantities.PowerSystemUnits.PU;
+
+import edu.ie3.datamodel.io.factory.EntityData;
+import edu.ie3.datamodel.models.result.CongestionResult;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Set;
+import javax.measure.quantity.Dimensionless;
+import tech.units.indriya.ComparableQuantity;
+
+public class CongestionResultFactory extends ResultEntityFactory<CongestionResult> {
+  private static final String SUBNET = "subnet";
```

Comment:
We've been using `grid` instead of `net` for some time now, so I think it'd make sense to use `subgrid` here as well

Reply:
The `NodeInput` is still using `net`.

Reply:
Yes, as in many other places. Should be changed at some point, but not here I'd say.

Reply:
Should I create a new issue for that?

Reply:
Would be great, thanks!